.azure-pipelines/integration-tests.yml

```diff
@@ -22,18 +22,18 @@ pool:
 variables:
   buildConfiguration: Release
   publishOutput: $(System.DefaultWorkingDirectory)/src/bin/managed-publish
-  dotnetCoreSdkVersion: 3.1.107
-  dotnetCoreSdk5Version: 5.0.100
+  dotnetCoreSdk5Version: 5.0.103
 
 jobs:
 
 - job: build_linux_profiler
 
   steps:
   - task: UseDotNet@2
-    displayName: install dotnet core sdk 3.1
+    displayName: install dotnet core runtime 3.1
```

Comment:
I always thought we only needed the latest SDK and only the runtimes for the rest, and it used to be like that, but at some point there was an issue (I can't remember) and we switched to installing all the SDK version instead. I hope this works as it will make the containers a bit smaller and faster to build, right?

Reply:
Oh, looks it was because the base runtime does not include aspnetcore runtime. See #868.

Reply:
Maybe not: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops#task-inputs

Reply:
Soo... yeah... you're right. However, it's _only_ [the Windows x64 job that fails currently](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=48304&view=logs&j=3c51c6d7-040c-5535-ae2f-42d5f8bb473b)... could be indicative that we're not running all the tests we think we are on other platforms?

As an aside, the Windows agents already have a bazillion versions installed, so we could probably get away without any of these anyway 🤷‍♂️

Reply:
I don't really understand why the OTel fork was able to make these changes in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/53 but we can't 🤔

Reply:
I think it works on Linux because we install the aspnetcore runtime separately in the dockerfile: https://github.com/DataDog/dd-trace-dotnet/blob/master/build/docker/dotnet.dockerfile

Reply:
argh, of course we do, I forget that this is only _part_ of the build definition 😛